Assets/Scripts/Controllers/Events/TradeController.cs

```diff
@@ -83,16 +92,36 @@ public void ShowTradeDialogBox(TraderShipController tradeShip)
         {
             tradeShip.TradeCompleted = true;
             TradeShips.Remove(tradeShip);
+            reservedLandingPads.Remove(tradeShip.LandingPad);
         };
         dbm.dialogBoxTrade.TradeCompleted = () =>
         {
             tradeShip.TradeCompleted = true;
             TrasfertTradedItems(trade, tradeShip.LandingCoordinates);
+            reservedLandingPads.Remove(tradeShip.LandingPad);
             TradeShips.Remove(tradeShip);
         };
         dbm.dialogBoxTrade.ShowDialog();
     }
 
+    public void CreateTradeJob(Furniture landingPad, Character playerBroker)
+    {
+        // First check if there's even a ship on the landing pad.
+        TraderShipController[] traders = TradeShips.Where(x => x.LandingPad == landingPad && x.DestinationReached).ToArray();
+        if (traders.Length != 1)
```

Comment:
Won't this if statement report that there is no trade ship on the landing pad if there are in fact two landing pads that both have trade ships?
so when length is 2, the function will error and bails